src/shared/utilities/textUtilities.ts

```diff
@@ -256,7 +256,7 @@ export function formatLocalized(d: Date = new Date()): string {
         hour: 'numeric',
         minute: 'numeric',
         second: 'numeric',
-        timeZoneName: 'longOffset',
+        timeZoneName: isCloud9() ? 'short' : 'shortOffset',
```

Comment:
Is there a reason to not use `short` for all?

Reply:
from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat

```
// short: 12/16/2021, PST
// long: 12/16/2021, Pacific Standard Time
// shortOffset: 12/16/2021, GMT-8
// longOffset: 12/16/2021, GMT-08:00
// shortGeneric: 12/16/2021, PT
// longGeneric: 12/16/2021, Pacific Time
```

I suspect that the offset format is more common in AWS, and perhaps more accessible: the shortname requires users to know all the timezone abbreviations, whereas the offset format only requires knowing "GMT".